.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Enhancement
+about: Propose an improvement to Rust.
+labels: C-enhancement
```

Comment:
Hm, shouldn't this be C-feature-request?

Reply:
Maybe, the line between the two is not clear to me.

Reply:
The intent (in my mind, maybe not everyone's :) is that C-enhancement should be landable without any team approval, e.g. a diagnostic fix. But a C-feature-request will likely need team member sign off (and likely eventually FCP). For example, adjusting documentation to clarify wording is likely just C-enhancement, but making a new guarantee would be C-feature-request.

Adding a new function to the standard library is also C-feature-request, since we want a libs team member to sign off on those prior to landing.

Reply:
Would it be better for that be another template then? Or should we not have one for `C-enhancement`?

Reply:
I think two templates make sense -- one for feature requests and one for enhancements. But maybe that's confusing for users, not sure. I'm not sure which one I'd prefer if we only have one.